### DISCUSSION TITLE
Integration of Facebook's ThreatExchange

### DISCUSSION THREAD

**Discussion by Jeremy-JJ** [qirtaiba/modtools#6] [Ideas]
Jeremy,

As a CS-IS major, well-aware of the ballooning problem of CSAM on user-driven social media websites, I've stumbled upon your project during my research into the development of CSAM and other safety moderation.

For starters, I'm impressed by your work and view any developments in this area to be of vital importance to the social stability of our specie, while maintaining our freedoms. It is of infinite importance we continue improving in this area to avoid both the social harm afflicted by CSAM, likewise the hammer of authoritarianism failure would invite.

As an aside, all of this has brought me to consider steering my major toward AI.

As for the subject of my inquiry, though I'm relatively new to these industry developments, is it possible to improve your project by integrating Facebook (Meta) ThreatExchange to add an additional layer?
https://developers.facebook.com/docs/threat-exchange
https://github.com/facebook/ThreatExchange/

Thanks for what you're doing,
Jeremy

**Comment by qirtaiba** [comment]
Thanks for your interest! Yes, this should absolutely be on the roadmap and has already been discussed. However, we don't currently have an active development team so this issue will need to stay open for now.
